burnman/eos/birch_murnaghan.py

```diff
@@ -139,28 +139,29 @@ def validate_parameters(self, params):
         Check for existence and validity of the parameters
         """
      
-        #if G and Gprime are not included this is presumably deliberate,
-        #as we can model density and bulk modulus just fine without them,
-        #so just add them to the dictionary as nans
+        if 'P_0' not in params:
+            params['P_0'] = 0.
+
+        # If G and Gprime are not included this is presumably deliberate,
+        # as we can model density and bulk modulus just fine without them,
+        # so just add them to the dictionary as nans
         if 'G_0' not in params:
             params['G_0'] = float('nan')
         if 'Gprime_0' not in params:
             params['Gprime_0'] = float('nan')
   
-        #check that all the required keys are in the dictionary
+        # Check that all the required keys are in the dictionary
         expected_keys = ['V_0', 'K_0', 'Kprime_0', 'G_0', 'Gprime_0']
         for k in expected_keys:
             if k not in params:
                 raise KeyError('params object missing parameter : ' + k)
         
-        #now check that the values are reasonable.  I mostly just
-        #made up these values from experience, and we are only 
-        #raising a warning.  Better way to do this? [IR]
+        #now check that the values are reasonable.
         if params['V_0'] < 1.e-7 or params['V_0'] > 1.e-3:
             warnings.warn( 'Unusual value for V_0', stacklevel=2 )
         if params['K_0'] < 1.e9 or params['K_0'] > 1.e13:
             warnings.warn( 'Unusual value for K_0' , stacklevel=2)
-        if params['Kprime_0'] < -5. or params['Kprime_0'] > 10.:
+        if params['Kprime_0'] < 0. or params['Kprime_0'] > 10.:
```

Comment:
Hmmm, it's a good question. I don't think there's any requirement that it must be positive, but a negative value would probably indicate a collapse in structure, and I doubt we'll have many users interested in such weird behaviour.

Actually, I changed this to zero because there was a test that checked for a negative K' in SLB/MGD. I think having this warning is useful, and the user will know when to ignore it if they have a strange material.